lusidtools/cocoon/cocoon.py

```diff
@@ -287,6 +287,47 @@ def load_instrument_property_batch(
 
         return results
 
+    @staticmethod
```

Comment:
this may have been there before, but why do these need to be `@staticmethod`s?

Reply:
I'm actually not sure. These were there before. @MikeMcGarry  - what do you think?

Reply:
They don't need to be static methods, but there is no reason for them to be class or instance methods.